From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@BrucePerlerMS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (bruceper). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

BrucePerlerMS commented:
ERROR ... this was not done with consensus of the DEV team.

BrucePerlerMS commented:
ERROR - this change needs to be reviewed by Dev team before I update the topic. And the Dev team needs to be in consensus about what's needed. :)

BrucePerlerMS commented:
sorry for the spam PR

Rajani-Janaki-Ram committed:
edit MVC

Rajani-Janaki-Ram committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Rajani-Janaki-Ram committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Rajani-Janaki-Ram committed:
mvc edits

Rajani-Janaki-Ram committed:
mvc edits

Rajani-Janaki-Ram committed:
mvc edits

Rajani-Janaki-Ram committed:
acrolinx

Rajani-Janaki-Ram committed:
acrolinx

Commit:
reoved tag details

Commit:
added hashes

Commit:
commited theirs

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Rajani-Janaki-Ram committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

barclayn committed:
fixing a problem with the example for querying data

paulathurman committed:
edit pass: data-factory-tutorials-batch1

Rajani-Janaki-Ram committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Rajani-Janaki-Ram committed:
fixes to docs

paulathurman committed:
fixing merge conflicts

paulathurman committed:
edit pass: data-factory-tutorials-batch1

paulathurman committed:
edit pass: data-factory-tutorials-batch2

liydu committed:
Add remote monitoring article and TOC node

liydu committed:
Update the result image

Lisaco88 committed:
Fixed typos in intro

Lisaco88 committed:
Fixed 2 typos

carolinacmoravia committed:
M22732: Grammar error: "probes and rules setup" vs "probes and rules are setup"

Hello, @rwike77, 

Translator has reported possible source content issue.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into data-factory-tutorials-batch2

paulathurman committed:
edit pass: data-factory-tutorials-batch2

alexwun committed:
Update application upgrade articles

Commit:
Fix indentation and correct image alts

Rajani-Janaki-Ram committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Rajani-Janaki-Ram committed:
fixes to docs

Rajani-Janaki-Ram committed:
Merge branch 'master' of https://github.com/Rajani-Janaki-Ram/azure-docs-pr

itechedit committed:
Update toc.yml

alexwun committed:
Minor fixes

paulathurman committed:
fixing merge conflicts

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into data-factory-tutorials-batch2

paulathurman committed:
edit pass: data-factory-tutorials-batch2

gatneil committed:
updated autoOS upgrades doc for recent changes

gatneil committed:
fixed link

gatneil committed:
updated image

gatneil committed:
updated image

tamram committed:
fix links

ShawnJackson committed:
Update tutorial-copy-data-portal.md

ShawnJackson committed:
Update tutorial-copy-data-tool.md

spelluru committed:
updated date

alexwun committed:
Remove dead link

paulathurman committed:
Update tutorial-copy-data-portal.md

paulathurman committed:
Merge branch 'master' into data-factory-tutorials-batch1

paulathurman committed:
Merge branch 'master' into data-factory-tutorials-batch2

paulathurman committed:
Update tutorial-hybrid-copy-data-tool.md

paulathurman committed:
Update tutorial-hybrid-copy-portal.md

paulathurman committed:
Update tutorial-copy-data-tool.md

AnithaAdusumilli committed:
update for SQL GA

ShawnJackson committed:
Merge pull request #32930 from paulathurman/data-factory-tutorials-batch1

edit pass: Data factory tutorials batch1

PRMerger13 committed:
Merge pull request #33024 from AnithaAdusumilli/patch-8

update for SQL GA

ShawnJackson committed:
Update tutorial-hybrid-copy-data-tool.md

ShawnJackson committed:
Update tutorial-hybrid-copy-portal.md

ShawnJackson committed:
Merge pull request #32937 from paulathurman/data-factory-tutorials-batch2

edit pass: Data factory tutorials batch2

gatneil committed:
updated for SF changes

gatneil committed:
update image

gatneil committed:
updated image

liydu committed:
Remove en-us in the URLs

PRMerger14 committed:
Merge pull request #32972 from tamram/tamram-0206

fix links in node.js qs

ktoliver committed:
Merge pull request #32967 from gatneil/autoOS

Update AutoOSUpgrade article for recent changes

ktoliver committed:
Merge pull request #32782 from liydu/az3166-samples-remote-monitoring

Az3166 samples remote monitoring

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Test0206-2

spelluru committed:
Fixed

Commit:
automation : update management

spelluru committed:
Author and Monitor

PRMerger19 committed:
Merge pull request #33031 from georgewallace/autoupdate

automation : update management

Sogup committed:
Update azure-backup-limits.md

spelluru committed:
updated screenshots

PRMerger16 committed:
Merge pull request #33030 from spelluru/SSIS0207

Ssis0207

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into onprem0207

spelluru committed:
Publish All

Commit:
Merge pull request #33032 from Sogup/patch-35

Update azure-backup-limits.md

JYOTHIRMAISURI committed:
updates to alerts-unified-usage article

JYOTHIRMAISURI committed:
format fix

JYOTHIRMAISURI committed:
capitalization - minor

JYOTHIRMAISURI committed:
update

christiankuhtz committed:
Tweak

Rajani-Janaki-Ram committed:
Merge branch 'master' into master

PRMerger16 committed:
Merge pull request #33038 from christiankuhtz/patch-53

Tweak

Rajani-Janaki-Ram committed:
fixes to docs

Rajani-Janaki-Ram committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Rajani-Janaki-Ram committed:
fixes to docs

linda33wj committed:
Clarify S3 required permission

linda33wj committed:
Clarify ADLS SP & MSI permission

linda33wj committed:
Clarify ADLA SP permission

PRMerger14 committed:
Merge pull request #33039 from linda33wj/master

Update ADF content

Commit:
adding IP address for East US

PRMerger17 committed:
Merge pull request #33042 from Blackmist/freshness

adding IP address for East US

PRMerger17 committed:
Merge pull request #32839 from carolinacmoravia/patch-596

M22732: Grammar error: "probes and rules setup" vs "probes and rules …

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
fixed hashes

Commit:
added more hashes

mrbullwinkle committed:
Clarify duration plus correct spelling error

PRMerger13 committed:
Merge pull request #33044 from ruturaj/master

fixed hashes

PRMerger18 committed:
Merge pull request #33045 from mrbullwinkle/mrb_02_08_2018_impact

Clarify duration plus correct spelling error

Lisaco88 committed:
Merge pull request #30517 from Rajani-Janaki-Ram/master

Master

Lisaco88 committed:
Merge pull request #33033 from spelluru/onprem0207

updated screenshots

alexwun committed:
Address PR comments

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

alexwun committed:
Fix table cell formatting

Commit:
Merge pull request #32653 from barclayn/asc-siem-fix

fixing a problem with the example for querying data

v-alje committed:
Merge pull request #33048 from MicrosoftDocs/FromPublicRepo

From public repo

Lisaco88 committed:
Merge pull request #32927 from alexwun/master

Update advanced application upgrade topics article

rmca14 committed:
Merge pull request #33034 from JYOTHIRMAISURI/master

updates to alerts-unified-usage article

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
